index.html

```diff
@@ -1,18 +1,184 @@
+      </svg>
+      <svg
+        class="ellipse-19 position-absolute"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <g filter="url(#filter0_f)">
+          <circle cx="84" cy="84" r="34" fill="#EDECFF" />
+        </g>
+        <defs>
+          <filter
+            x="0"
+            y="0"
+            width="168"
+            height="168"
+            filterUnits="userSpaceOnUse"
+            color-interpolation-filters="sRGB"
+          >
+            <feFlood flood-opacity="0" result="BackgroundImageFix" />
+            <feBlend
+              mode="normal"
+              in="SourceGraphic"
+              in2="BackgroundImageFix"
+              result="shape"
+            />
+            <feGaussianBlur stdDeviation="25" result="effect1_foregroundBlur" />
+          </filter>
+        </defs>
+      </svg>
+
+      <h1 class="name-header">Basit Korede</h1>
+      <section class="about-me-section">
+        <h2 class="about-me-header">Software Developer</h2>
+
+        <p>
+          I can help you build a product,feature or website look through some of
+          my work and experience! if you like what you see and have a project
+          you need coded, don't hesitate to contact me.
+        </p>
+      </section>
+      <section aria-label="specializations" class="specialization-section">
+        <ul
+          class="specialization-list display-flex justify-content-space-between"
+        >
+          <li class="specialization-li">
+            language
+            <ul class="specialization-li-ul">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+          </li>
+          <li class="specialization-li">
+            frameworks
+            <ul class="specialization-li-ul">
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+          </li>
+          <li class="specialization-li">
+            skills
+            <ul class="specialization-li-ul">
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </li>
+        </ul>
+      </section>
 
-</body>
+      <div class="social-media-div">
+        <ul class="social-media-ul display-flex justify-content-center">
+          <li class="social-media-li">
+            <a
+              ><img
+                src="./images/github-icon.png"
+                class="social-media-image"
+                alt="github icon"
+            /></a>
+          </li>
 
-</html>
\ No newline at end of file
+          <li class="social-media-li">
+            <a
+              ><img
+                src="./images/linkdin.svg"
+                class="social-media-image"
+                alt="linkdin icon"
+            /></a>
```

Comment:
All decorations should always be background images and not <img /> tags. Please refer to this [article](https://www.w3docs.com/snippets/html/when-to-use-html-img-tag-and-css-background-image-property.html) for more on this. Let us fix this in every other case